layouts/index.html

```diff
@@ -9,14 +9,14 @@
                             <a class="homepage-brand" href="/about">
                                 <img class="img-responsive" alt="Nats.io" src="/img/large-logo.png">
                             </a>
-                            <h1><span class="nats-colour-blue-lite">Open Source.</span> <span class="nats-colour-green-dark">Simple.</span> <span class="nats-colour-blue-dark">Secure.</span> <span class="nats-colour-green-lite">Scalable.</span></h1>
+                            <h1><span class="nats-colour-blue-lite">Secure.</span> <span class="nats-colour-green-dark">Simple.</span> <span class="nats-colour-blue-dark">Scalable.</span> <span class="nats-colour-green-lite">Open Source.</span></h1>
                             {{ .Content }}
 
-                            <p class="bottom-info">NATS Server is a simple, high performance open source messaging system for cloud native applications, IoT messaging, and microservices architectures.</p>
+                            <p class="bottom-info">NATS.io is a simple, secure and high performance open source messaging system for cloud native applications, IoT messaging, and microservices architectures.</p>
 
-                            <p class="bottom-info">We are a Cloud Native Computing Foundation member project. Members of the Synadia team created the NATS Server (written in Go), NATS Streaming, and clients written in Python, Ruby, Node.js, Elixir, Java, NGINX, C, and C#. The community has contributed a growing list of libraries, including Arduino, Rust, Lua, PHP, Perl, and <a href="/download">more</a>.</p>
-
-                            <p class="bottom-info">Companies like OpenTable, Ericsson, HTC, Siemens, VMware, Pivotal, GE and Baidu rely on the NATS enterprise messaging system for its highly performant and resilient capabilities.</p>
+                            <p class="bottom-info">We are a href="https://cncf.io/projects">Cloud Native Computing Foundation</a> project. Members of the Synadia team created and maintain the NATS and Streaming Servers, as well as clients
+                            written in Go, Java, Javascript/Typescript, Python, Ruby, C#, C, and NGINX. The community has also contributed a growing list of client libraries, including Arduino, Rust, Lua, PHP, Perl, and <a href="/download">more</a>.</p>
```

Comment:
I believe we are supposed to specify we are an 'incubating project' in the CNCF.n

Reply:
The link goes to the CNCF page that shows our status.